docs/guide/getting-started.md

```diff
@@ -31,26 +31,13 @@ $ yarn add --dev vitepress vue
 ::: details Getting missing peer deps warnings?
 `@docsearch/js` has certain issues with its peer dependencies. If you see some commands failing due to them, you can try this workaround for now:
 
-On Yarn v2/v3, add this inside your rc file (`.yarnrc.yml` by default):
```

Comment:
Unlike PNPM, Yarn doesn't hard fail when peer deps aren't met properly. Also, this workaround no longer prevent all warnings. In some versions `@algolia/client-search` was not required, there it worked. But its broken again. So I'm removing this for now.